front/src/layouts/footer/footer.html

```diff
@@ -0,0 +1,43 @@
+<div class="body main-color">  
+  <div class="color-background main">  
+    <div class="logo">
+      <logo/>
+      <div class="stores">
+        <el-link>
+          <img alt="App Store" src="../../assets/appStore.png">
+        </el-link>
+        <el-link>
+          <img alt="Google play" src="../../assets/googlePlay.png">
+        </el-link>       
+      </div>    
+    </div>
+    <span>
+      <el-button class="color-background main-color footer">Фильмы</el-button>
+      <el-button class="color-background main-color footer">Сериалы</el-button>
+      <el-button class="color-background main-color footer">Мультфильмы</el-button>
+      <el-button class="color-background main-color footer">Новости</el-button>
+      <el-button class="color-background main-color footer">Актеры</el-button>      
+    </span>
+    <div class="socialNetworks">
+      <div>Социальные сети</div>
+      <el-link>
+        <img alt="Google" src="../../assets/google.png">
+      </el-link>
+      <el-link>
+        <img alt="Twitter" src="../../assets/twitter.png">
+      </el-link>
+      <el-link>
+        <img alt="Facebook" src="../../assets/facebook.png">
+      </el-link>
+    </div>   
+  </div>
+  <div class="lowerPart footer-color-background">
+    <span>
+      (с) Все права защищены
+    </span>
+    <el-button class="footer-color-background main-color footer">Пользовательское соглашение</el-button>
+    <el-button class="footer-color-background main-color privicyPolice">Privicy Police</el-button>
```

Comment:
why buttons? if it is link to some pages it should be link. it doesn't have any common with buttons